src/Database/Query.php

```diff
@@ -1605,7 +1605,7 @@ public function newExpr($rawExpression = null)
      */
     public function func()
     {
-        if (empty($this->_functionsBuilder)) {
+        if ($this->_functionsBuilder === null) {
```

Comment:
should be okay, but maybe assign `null` up above. Could use falsey but I find that to be a variant on the empty.